interpreter/src/main/scala/pl/touk/nussknacker/engine/ModelData.scala

```diff
@@ -94,7 +94,12 @@ trait ModelData extends BaseModelData with AutoCloseable {
   lazy val dictServices: UiDictServices =
     DictServicesFactoryLoader.justOne(modelClassLoader.classLoader).createUiDictServices(processDefinition.expressionConfig.dictionaries, processConfig)
 
-  lazy val validator: ProcessValidator = ProcessValidator.default(processWithObjectsDefinition, dictServices.dictRegistry, modelClassLoader.classLoader)
+  def prepareValidatorForCategory(category: String): ProcessValidator =
```

Comment:
maybe let's have category: Option[String]? For most backend tests we don't really care about category?